tasks/main.yml

```
@@ -627,6 +627,193 @@
         __mssql_conf_setting: "network forceencryption"
         __mssql_conf_setting_value: "{{ '1' if mssql_tls_enable else 'unset' }}"
 
+- name: Configure Active Directory authentication
+  vars:
+    __mssql_kinit_user: >-
+      {{ ad_integration_user }}@{{ ad_integration_realm
+      | upper }}
+  when: mssql_ad_configure | bool
+  block:
+    - name: Join to {{ ad_integration_realm }} realm
```

```suggestion
    - name: Join to realm {{ ad_integration_realm }}
```
ansible-lint 6.x - jinja templates must be at the end of `name` values